src/common/Database/MySQLConnection.h

```diff
@@ -89,13 +91,13 @@ class MySQLConnection
         {
             /// Tries to acquire lock. If lock is acquired by another thread
             /// the calling parent will just try another connection
-            return m_Mutex.tryacquire() != -1;
+            return m_Mutex.lock() != -1;
```

Comment:
supposed to be https://en.cppreference.com/w/cpp/thread/mutex/try_lock ?

Reply:
Yes you right! its just copy-paste because i falling asleep on the move and I just put what was in the buffer in order to fix it.

Reply:
still not right because try_lock returns boolean, not an int like ACE.

Reply:
Hm but i as i see no int types for mutes try_lock in std

Reply:
you simply `return m_Mutex.try_lock();`.

Reply:
@Rochet2 thanks